src/main/java/com/faforever/client/main/MainController.java

```diff
@@ -338,6 +339,20 @@ public void display() {
     registerWindowListeners();
   }
 
+  private void hideSplashScreen() {
```

Comment:
I suggest to make variables final wherever possible. Here is some explanation why:
http://www.codeaffine.com/2014/04/28/java-code-style-the-final-decision/
I also suggest to make the method static to make clear that it's standalone and has no object dependencies.